python/requirements/ml/requirements_dl.txt

```
@@ -4,12 +4,13 @@
 tensorflow==2.6.2
 tensorflow-probability==0.14.1
 
-# If you make changes to the torch versions below, please also make the corresponding changes to `requirements_ml_docker.txt`!
+# If you make changes to the versions below, please also make the corresponding changes to `requirements_ml_docker.txt`!
 
-torch==1.9.0;sys_platform=="darwin"
-torchvision==0.10.0;sys_platform=="darwin"
-
-# On non-OSX machines only install CPU version of torch and torchvision
--f https://download.pytorch.org/whl/torch_stable.html
-torch==1.9.0+cpu;sys_platform!="darwin"
-torchvision==0.10.0+cpu;sys_platform!="darwin"
+--extra-index-url https://download.pytorch.org/whl/cpu  # for CPU versions of torch, torchvision
+--find-links https://data.pyg.org/whl/torch-1.9.0+cu111.html  # for GPU versions of torch-scatter, torch-sparse ref https://github.com/ray-project/ray/pull/26072#discussion_r929330581
+torch==1.9.0
+torchvision==0.10.0
+torch-scatter==2.0.9
+torch-sparse==0.6.12
```

@amogkam from their [docs](https://pytorch-geometric.readthedocs.io/en/2.0.4/notes/installation.html), should we also add `torch-cluster` and `torch-spline-conv`?

```suggestion
--find-links https://data.pyg.org/whl/torch-1.9.0+cu111.html  # for GPU versions of torch-scatter, torch-sparse, torch-cluster, torch-spline-conv ref https://github.com/ray-project/ray/pull/26072#discussion_r929330581
torch==1.9.0
torchvision==0.10.0
torch-scatter==2.0.9
torch-sparse==0.6.12
torch-cluster==1.5.9
torch-spline-conv==1.2.1
```
I guess not necessary as CI was green, but feel free to commit the suggestion :)

---

[bd23199](https://github.com/ray-project/ray/pull/26072/commits/bd23199fdffd206a5af1d14874cfafce6fde32f0)